tests/unit_node/http_test.ts

```diff
@@ -993,7 +1003,8 @@ Deno.test(
     );
     const { promise, resolve, reject } = Promise.withResolvers<void>();
 
-    const request = http.request("http://localhost:5929/");
+    // TODO(kt3k): This should be "localhost" when we implemented happy-eyeballs for node:net
+    const request = http.request("http://127.0.0.1:5929/");
```

Comment:
Will all these changes cause regressions in already working libraries? Should we implement happy-eyeballs first?

Reply:
> Should we implement happy-eyeballs first?

I guess that is safer.